### DISCUSSION TITLE
Love this repo - Request to modify styling

### DISCUSSION THREAD

**Discussion by ItsJustPW** [Power-Components/livewire-powergrid#201] [Q&A]
Really great work on the data table, but I'd really like to know if it's at all possible to do the following: 

- Change the header background and text colour?
- Ability to have 'pills' as seen here: [Screenshot](https://camo.githubusercontent.com/c7c48df2793354749a31e8943073132a3139f88919fcc8e70ccfb207b0af2ca5/68747470733a2f2f696d6775722e636f6d2f516f456443376e2e706e67)

Any guidance if this is possible would be much appreciated.

**Comment by devsquad-luan-freitas** [comment]
Hello thank you so much. It helps ?

**headerAttribute** - https://livewire-powergrid.docsforge.com/main/include-columns/#headerattributestring-class-string-style

**bodyAttribute** - https://livewire-powergrid.docsforge.com/main/include-columns/#bodyattributestring-class-string-style
